metpy/calc/kinematics.py

```diff
@@ -748,3 +748,95 @@ def absolute_vorticity(u, v, dx, dy, lats, dim_order='yx'):
     f = coriolis_parameter(lats)
     relative_vorticity = vorticity(u, v, dx, dy, dim_order=dim_order)
     return relative_vorticity + f
+
+
+@exporter.export
+@check_units('[speed]', '[speed]', '[length]', '[length]', '[dimensionless]',
+             '[temperature]', '[pressure]')
+def potential_vorticity_baroclinic(u, v, dx, dy, lats, potential_temperature, pressure,
+                                   axis=0, dim_order='yx'):
+    r"""Calculate the baroclinic potential vorticity in a layer.
```

Comment:
In a layer?

Reply:
We do require multiple levels, but since it’s not really an average, I agree.